index.md

```diff
@@ -64,3 +64,4 @@ As a brief summary, changes include
 * ` /tools/{id}/versions/{version_id}/{type}/files` endpoint added to describe all avaliable files for a tool
 * checker workflows ([26](https://github.com/ga4gh/tool-registry-service-schemas/pull/26))
 * process for auto-generation of OpenAPI 3 copy of the schema
+* removal of service-info
```

Comment:
Maybe add that it was intended to be added back in 2.0.1

Reply:
Changed. Mentioned /metadata endpoint since V1 never had a /service-info endpoint (V2 added then removed it)